core/jazz_metrics/components/utils.js

```diff
@@ -62,14 +63,21 @@ function assetData(results, assetItem) {
   return asset_obj;
 };
 
-function getNameSpaceAndMetricDimensons(nameSpaceFrmAsset) {
+function getNameSpaceAndMetricDimensons(nameSpaceFrmAsset, provider) {
   var output_obj = {};
   output_obj["isError"] = false;
   var paramMetrics = [];
   var nameSpace = nameSpaceFrmAsset.toLowerCase();
   var namespacesList = metricConfig.namespaces;
-  var supportedNamespace = namespacesList[nameSpace];
-  if(nameSpaceFrmAsset && supportedNamespace) {
+
+  if(!namespacesList[provider]) {
+    output_obj["isError"] = true;
+    output_obj["nameSpace"] = `Provider not defined for namespace: ${nameSpace}`;
```

Comment:
Can we use 'message' key when isError = true?

Reply:
updated